build/cucumber-report.json

```diff
@@ -0,0 +1,221 @@
+[
```

Comment:
I'm not sure. The cucumber-report.json is used to help produce html reports, which give user-friendly output. It's not something that we are able to make use of currently, due to the fact that the reports only live in the temporary pod and then disappear as soon as the tests have finished running. However, it might be that there is a way of copying them during the run or something like that. EIther way, I think that they might need to be in git, but perhaps they don't belong in the build directory then?